test/node/serde.js

```diff
@@ -4,7 +4,7 @@ const tap = require('tap');
 
 const jstp = require('../..');
 
-const testCases = require('../fixtures/test-cases');
+const testCases = require('../fixtures/serde.test-cases');
```

Comment:
I don't want this to be `serde-test-cases`, because `serde` is like a namespace and it shouldn't be separated the same character used for separating words, IMHO.